STMReco/src/SConscript

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+#
+# Script to build the files found in this directory.
+#
+# Original author Rob Kutschke.
+#
+
+import os, re
+Import('env')
+Import('mu2e_helper')
+
+helper=mu2e_helper(env);
+rootlibs  = env['ROOTLIBS']
+
+rootlibs = env['ROOTLIBS']
+
+mainlib = helper.make_mainlib ( [ 'art_Persistency_Provenance',
+                                  'art_Persistency_Common',
+                                  'art_root_io_TFileService',
+                                  'art_Framework_Services_Registry',
+                                  'art_root_io_tfile_support',
+                                  'art_Utilities',
+                                  'canvas',
+                                  'cetlib',
+                                  'cetlib_except',
+                                  'CLHEP',
+                                  rootlibs
+                                ] )
+
+helper.make_plugins( [ mainlib,
+                       'mu2e_Mu2eUtilities',
+                       'mu2e_GeometryService',
+                       'mu2e_GlobalConstantsService',
+                       'mu2e_ConfigTools',
+                       'mu2e_Mu2eInterfaces',
+                       'mu2e_DataProducts',
+                       'mu2e_GeneralUtilities',
+                       'art_Framework_Core',
+                       'art_Framework_Principal',
+                       'art_Framework_Services_Registry',
+                       'art_root_io_tfile_support',
+                       'art_root_io_TFileService',
+                       'art_Persistency_Common',
+                       'art_Persistency_Provenance',
+                       'art_Utilities',
+                       'canvas',
+                       'MF_MessageLogger',
+                       'fhiclcpp',
+                       'fhiclcpp_types',
+                       'tbb',
+                       'cetlib',
+                       'cetlib_except',
+                       'CLHEP',
+                       'boost_filesystem',
+                       rootlibs,
+                       'pthread'
+                     ] )
+
+# helper.make_dict_and_map( [
```

Comment:
I prefer that you remove the commented code.  If and when you might want to re-enable it, the pattern might have changed.  So you should start from a known working example.